ci-operator/step-registry/ipi/conf/azure/confidential/trustedlaunch/ipi-conf-azure-confidential-trustedlaunch-commands.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+if [[ -z "${SECURITY_TYPE}" ]] || [[ "${SECURITY_TYPE}" != "TrustedLaunch" ]];  then
```

Comment:
Based on we already indicate the step is for `trustedlaunch` type, so I think we does not need this condition any more?